qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -38,12 +36,17 @@ def __init__(
         initial_state: Optional[QuantumCircuit] = None,
         mixer: Union[QuantumCircuit, OperatorBase] = None,
         initial_point: Optional[np.ndarray] = None,
+        alpha: float = 1.0,
```

Comment:
Since the argument is not a keyword-only argument, it should be added last for backward compatibility.

Reply:
Since the QAOAClient is new in this release we do not need to worry about backwards compatibility.

Reply:
Oh sorry. I identified it with VQERuntime.